back/engines/free/email_campaigns/app/models/concerns/email_campaigns/content_configurable.rb

```diff
@@ -7,14 +7,25 @@ module ContentConfigurable
     included do
       validates :subject_multiloc, presence: true, multiloc: {presence: true, length: {maximum: MAX_SUBJECT_LEN}}
       validates :body_multiloc, presence: true, multiloc: {presence: true}
-
       after_save :process_body_images
+
+      before_validation :sanitize_body_multiloc
     end
 
     def process_body_images
       processed_body_multiloc = TextImageService.new.swap_data_images(self, :body_multiloc)
       self.update_column(:body_multiloc, processed_body_multiloc)
     end
 
+    def sanitize_body_multiloc
+      service = SanitizationService.new
+      self.body_multiloc = service.sanitize_multiloc(
+        self.body_multiloc,
+        %i{title alignment list decoration link image video}
+      )
+      self.body_multiloc = service.linkify_multiloc(self.body_multiloc)
+      self.body_multiloc = service.remove_multiloc_empty_trailing_tags(self.body_multiloc)
+    end
+
```

Comment:
I think this is fine as, very likely.

You can test it out by watching the preview in the app, after composing your email. The HTML rendered there will be the sanitized version. If it comes out as intended, it's good. Would indeed expect it to be.

Reply:
Will do an epic deployment to test just in case 👍